java/src/jmri/jmrit/symbolicprog/FnMapPanelESU.java

```diff
@@ -191,7 +191,7 @@ public FnMapPanelESU(VariableTableModel v, List<Integer> varsUsed, Element model
             itemDescESU[item++] = Bundle.getMessage("FnMap_HL") + "[2]";
             itemDescESU[item++] = Bundle.getMessage("FnMap_RL") + "[2]";
             for (int i = 1; i <= 2; i++) {
-                itemDescESU[item++] = Bundle.getMessage("FnMap_A") + " " + i + (i <= 2 ? "[2]" : "");
+                itemDescESU[item++] = Bundle.getMessage("FnMap_A") + " " + i + "[2]";
```

Comment:
@rhwood Checked and all is well. Thanks. Must have missed that one. I have been running FindBugs on code when I make changes.